.github/workflows/intergation-tests.yml

```diff
@@ -0,0 +1,30 @@
+name: "Trigger Integration tests"
+on:
+  push:
+    branches:
+      - master
+      - 'releases/*'
+  pull_request:
+    branches:
+      - master
+      - 'releases/*'
+jobs: 
+    trigger-integration-tests:
+      name: Trigger Integration tests
+      runs-on: ubuntu-latest
+      steps:
+        - name: Extract branch name
+          id: extract_branch
+          run: |
+            echo "##[set-output name=branchname;]$(echo ${GITHUB_REF##*/})"
+
+        - name: Trigger Test run
+          if: github.event.pull_request.base.ref == 'releases/v1' || steps.extract_branch.outputs.branchname == 'releases/v1' || github.event.pull_request.base.ref == 'master' || steps.extract_branch.outputs.branchname == 'master'
+          run: |
+           response=$(curl -X POST -H "Authorization: token ${{ secrets.L2_REPO_TOKEN }}" https://api.github.com/repos/Azure/azure-actions-integration-tests/dispatches --data '{"event_type": "K8sCreateSecretReleasesPR", "client_payload": {"action": "CreateSecret", "commit": "${{ github.event.pull_request.head.sha }}", "repository":  "${{ github.repository }}", "prNumber": "${{ github.event.pull_request.number }}", "tobranch": "${{ github.event.pull_request.base.ref }}",  "frombranch": "${{ github.event.pull_request.head.ref }}" }}')
```

Comment:
I don't like the entire inline thing.. it's hard to read.

Reply:
Moved to a file.